src/components/Projects/Projects.js

```diff
@@ -0,0 +1,68 @@
+                {id: 3, url: url7, alt: 'Zdjęcie aplikacji - Because I love coffee...', isActive: true, info: 'Aplikacja do oceniania wrocławskich kawiarni -', title: '"Because I love coffee..."', isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} JS + Local Storage ${check} API Zomato ${check} Express.js ${check} Webpack 4`, live: 'https://angelika7.github.io/repo-search-cafes/', github: 'https://github.com/angelika7/repo-search-cafes'},
+                {id: 4, url: url6, alt: `Zdjęcie aplikacji - ${appTitle}`, isActive: true, info: 'Kieszonkowa aplikacja pogodowa -', title: `"${appTitle}"`, isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) + SVG ${check} JS ${check} API Open Weather Map`, live: 'https://angelika7.github.io/weather-app/', github: 'https://github.com/angelika7/weather-app'},
+                {id: 5, url: url5, alt: 'Zdjęcie aplikacji - Portfolio v1.0', isActive: true, info: 'Strona internetowa -', title: '"Portfolio v1.0"', isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} JS ${check} Express.js ${check} Grunt.js`, live: 'https://my-portfolio-cv.herokuapp.com/', github: 'https://github.com/angelika7/portfolio'},
+                {id: 6, url: url4, alt: 'Zdjęcie aplikacji - Manufaktura słodyczy', isActive: true, info: 'Strona internetowa cukierni -', title: '"Manufaktura słodyczy"', isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} JS + jQuery`, live: 'https://angelika7.github.io/cukiernia-repo/', github: 'https://github.com/angelika7/cukiernia-repo'},
+                {id: 7, url: url3, alt: 'Zdjęcie aplikacji - In the woods', isActive: true, info: 'Strona internetowa o fotografii (PSD to HTML) -', title: '"In the woods"', isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} jQuery + WOW.js ${check} Bootstrap 4`, live: 'https://angelika7.github.io/photography-website-repo/', github: 'https://github.com/angelika7/photography-website-repo'},
+                {id: 8, url: url2, alt: 'Zdjęcie aplikacji - Lets get lost', isActive: true, info: 'Strona internetowa o podróżach (PSD to HTML) -', title: '"Lets get lost"', isFlipped: false, backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} jQuery + WOW.js ${check} Grunt.js`, live: 'https://angelika7.github.io/travelling-website-repo/', github: 'https://github.com/angelika7/travelling-website-repo'},
+                {id: 9, url: url1, alt: 'Zdjęcie aplikacji - salon fryzjerski - Wiktoria', isActive: true, info: 'Strona internetowa salonu fryzjerskiego -', title: '"Wiktoria"', backInfo: `${check} HTML5 ${check} CSS3 (SCSS) ${check} JS`, isFlipped: false, live: 'https://angelika7.github.io/salon-repo/', github: 'https://github.com/angelika7/salon-repo'}
+            ],
+        }
+        this.onClickHandler = this.onClickHandler.bind(this);
```

Comment:
there is a lot of inconsistency in your codebase. Either stick to function components of class components that are not using bind or to ones that do so through the repository. 

Keeping the codebase unified will ease the reading experience of another developer. 

Making code readable is one of the most important skills under your belt ;) 

This comment also applies to Stack.js file